lib/aytests/image_builder.rb

```diff
@@ -65,7 +66,7 @@ def install(autoinst, iso_url)
       setup_iso(iso_url)
       setup_autoinst(autoinst)
       setup_definition(:install)
-      build
+      build(build_environment(autoinst))
```

Comment:
I like it better now. But I actually wanted to go one step further in the same direction having
```
build(autoinst)
```